modules/ROOT/pages/_partials/task-configure-proxy.adoc

```diff
@@ -0,0 +1,72 @@
+//tag::first-steps[]
+. In API Manager, click *Add API* and select *Add new API*.
+. Select *Mule Gateway* as your runtime.
+. Select *Deploy a proxy application* for *Proxy type*.
+//end::first-steps[]
+//tag::target-type[]
+. Select your *Target type* from the following options: CloudHub, Hybrid, or Runtime Fabric.
+. Select the appropriate parameters for your target type.
++
+The parameter options for your target will differ depending on the target type you select.
+//end::target-type[]
+//tag::hybrid[]
+. Select *Hybrid* for *Proxy deployment target*, then select the target server.
+//end::hybrid[]
+//tag::mid-steps[]
+. Click *Next*.
+. Select an API from the following options:
+* Click **Select API from Exchange** if you have an API shared with you through Exchange that you want to manage.
+[arabic]
+.. Click the API from the list under **Select API**. You can search for a specific API if needed.
+.. Update the Asset type, API version, and Asset version if needed.
+
+* Click **Create new API**.
+[arabic]
+.. Enter a name for the new API asset.
+//end::mid-steps[]
+//tag::asset-type-options[]
+.. Select the **Asset type** from the following options:
+
+** **REST API:** Select REST API if you have an RAML or OAS API definition file you want to include for your asset.
```

Comment:
```suggestion
** **REST API:** Select REST API if you have a RAML or OAS API definition file you want to include for your asset.
```

Reply:
will fix

Reply:
One more comment on this - you probably should bold  "REST API" per our style (Select *REST API*...). However, that ends up being a lot of bold in the sentence. To avoid this, I suggest "Select this option..." here and wherever else it's applicable.